### ISSUE TITLE
Ability to store violation error message

### ISSUE THREAD

**Issue by not-relevant-is-it** [liampauling/flumine#297]
In `flumine.controls.__init__.py`, `BaseControl._on_error()` logs error message then it is gone, something like
```
class BaseControl:
    def _on_error(self, order: BaseOrder, error: str) -> None:
        order.violation_message = error
        ....
   ...
```
to store error?

**Comment by liampauling** [comment]
Like this?

https://github.com/liampauling/flumine/pull/298

**Comment by not-relevant-is-it** [comment]
yeah, it is the specific `error` string, looks like it's covered in the commit where the message is passed to the violation function `order.violation(violation_msg)`. Do you know when that branch/task will be added to master? I am fairly new to Github so not 100% sure how that works

**Comment by liampauling** [comment]
Will get a release out this evening or tomorrow am

**Comment by liampauling** [comment]
Released in 0.14.1
